### PR TITLE
Fix generating default timestamp

### PR DESCRIPTION
**Pull request by andreban**
- Generates default timestamp.
- Updates version to 0.1.1

### CODE REVIEW

File: update-cache/lib/UpdateCacheUrlProvider.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ class UpdateCacheUrlProvider {
    * @returns {Array<Object>} an array with objects containing the cache ID, cache name and
    * update-cache url.
    */
-  calculateFromOriginUrl(originUrl, timestamp) {
+  calculateFromOriginUrl(originUrl, timestamp = defaultTimestamp_()) {
```
Comment (sebastianbenz) [inline update-cache/lib/UpdateCacheUrlProvider.js]: Can you also update the JS DOC to mark it as optional.

File: update-cache/lib/UpdateCacheUrlProvider.js [hunk 2]
```diff
@@ -60,7 +60,7 @@ class UpdateCacheUrlProvider {
    * @param {Number} timestamp as a UNIX Epoch in seconds
```
Comment (sebastianbenz) [inline update-cache/lib/UpdateCacheUrlProvider.js]: Mark as optional

Comment (andreban) [reply update-cache/lib/UpdateCacheUrlProvider.js]: done.

File: update-cache/lib/UpdateCacheUrlProvider.js (hunk 1 above)
Comment (andreban) [reply update-cache/lib/UpdateCacheUrlProvider.js]: done.

**Pull request merged**